bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/TelldusLiveBridgeHandler.java

```diff
@@ -51,14 +53,19 @@
  */
 public class TelldusLiveBridgeHandler extends BaseBridgeHandler implements TelldusBridgeHandler {
 
+    private static final int REFRESH_DELAY = 10;
+
     private final Logger logger = LoggerFactory.getLogger(TelldusLiveBridgeHandler.class);
 
     private TellstickNetDevices deviceList = null;
     private TellstickNetSensors sensorList = null;
     private TelldusLiveDeviceController controller = new TelldusLiveDeviceController();
     private List<DeviceStatusListener> deviceStatusListeners = new Vector<>();
 
-    private static final int REFRESH_DELAY = 10;
+    private int nbRefresh;
+    private long sumRefreshDuration;
+    private long minRefreshDuration = 1000000;
```

Comment:
Consider for readability:
```suggestion
    private long minRefreshDuration = 1_000_000;
```

Reply:
Done